docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
```

Comment:
The title says "working group" but this says "workgroup" -- appreciate your point on consistency in the PR body, but it feels odd to leave it unadressed in a copyediting pass such as this.

Reply:
I do _hope_ to address this here; per our discussion on Discord as well as mentioned here, I'm just hoping to get some clarity on nomenclature first before going through and changing it everywhere.

The problem is the PSF itself is not at all consistent on this. Just on their [main workgroup page](https://www.python.org/psf/committees/) alone, they use "workgroups" in the title, "work group" in the top level headings and "working group" in many of the resolutions and descriptions (and both workgroup and work group in others). In total, "work group" is used 40 times (mostly in the headings), "workgroup" 15 times and "working group" 12 times. Plus "committee" is used 46 times, aside from in the URL itself, mostly interchangeably with the previous three.

"Workgroup" (and WG) is used on the [example workgroup page](https://wiki.python.org/psf/Example%20PSF%20Workgroup%20Page) and the [example charter](https://wiki.python.org/psf/Example%20PSF%20Workgroup%20Charter), but the title of [every "workgroup" page](https://wiki.python.org/psf/Contents#id4) uses "Working Group". and the charters and info pages use a mix of all three. Ughhhhhh...